src/test/java/seedu/academyassist/logic/commands/AddCommandTest.java

```diff
@@ -153,12 +154,12 @@ public boolean hasPersonWithStudentId(StudentId studentId) {
         }
 
         @Override
-        public boolean personDuplicateClass(Subject subject, Person student) {
+        public boolean personDuplicateClass(Set<Subject> subjects, Person student) {
             throw new AssertionError("This method should not be called.");
         }
 
         @Override
-        public void addSubjectToPerson(Subject subject, Person person) {
+        public void addSubjectsToPerson(Set<Subject> subjects, Person person) {
```

Comment:
Good that multiple subjects can be added at the same time which improves efficiency